### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by jafaripur**
Update dependency and support ext-amqp version 2.0.0

### CODE REVIEW

**Comment by makasim** [comment]
Have you tested it ?

**Comment by jafaripur** [comment]
Yes, with cloning the project and change autoloading class PSR name in composer.json (for bypass composer).

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.4|^8.0",
-        "ext-amqp": "^1.9.3",
+        "ext-amqp": "^1.9.3|^2.0.0",
```
Comment (makasim) [inline composer.json:10]: pkg/amqp-ext/composer.json should be updated too

**Comment by makasim** [comment]
Thanks for your pull request! We love contributions.

However, this repository is what we call a "subtree split": a read-only copy of one directory of the main enqueue-dev repository. It is used by Composer to allow developers to depend on specific Enqueue package.

If you want to contribute, you should instead open a pull request on the main repository:

https://github.com/php-enqueue/enqueue-dev

Read the contribution guide

https://github.com/php-enqueue/enqueue-dev/blob/master/docs/contribution.md

Thank you for your contribution!

**Pull request closed without merging**